### ISSUE TITLE
Trans Matrix regularize seems wrong

### ISSUE THREAD

**Issue by xiaozai** [fxia22/pointnet.pytorch#56]
Hi , In train_classification.py, it used the `loss += feature_transform_regularize(trans_feat)*0.001', it should be trans, not trans_feat, trans is the transfromation matrix which needs to be regularized.
